2-Browsers/Week1/homework/ex1-bookList/index.js

```diff
@@ -18,8 +18,88 @@
 */
 //cspell: enable
 
+class DomElement {
```

Comment:
Nice that you know how to create an abstraction like this class for DOM elements. I don't think I would do this this in practice though.